packages/blog-starter-kit/themes/hashnode/components/analytics.tsx

```diff
@@ -4,18 +4,20 @@ import { v4 as uuidv4 } from 'uuid';
 
 import { useAppContext } from './contexts/appContext';
 const GA_TRACKING_ID = 'G-72XG3F8LNJ'; // This is Hashnode's GA tracking ID
+const HASHNODE_ANALYTICS_URL = 'https://user-analytics.hashnode.com/api/analytics'; // This is Hashnode's internal analytics URL
```

Comment:
Can we also use an env variable instead and proxy it like we did with `/api/collect`. Also, the proxy for `/api/collect` and the env var for that should be removed.

Reply:
Proxied but kept it in variable instead of env. This is not something that will change per user basis so no need to keep it in env. Analytics URL will be same always.